File: exo_11.exs
# Pas fonctionnel, faut que j'apprennent comment marchent les loops plus en détail

year = IO.gets("En quelle année es-tu né(e) ?\n> ") |> String.trim("\n")
i = String.to_integer(year)
date = DateTime.utc_now() |> Map.fetch!(:year)
delta = date - i
y = 0

cond do
  String.length(year) == 0 ->
    IO.puts("Et ton année de naissance ?\n")

  true ->
      for x <- delta..0,
      i <= date,
      do:
        (IO.puts("Il y a #{x} ans, tu avais #{y} ans"); y = y + 1)
end

File: exo_11.rb
print "En quelle année es-tu né(e) ?\n> "
year = gets.chomp
i = year.to_i
x = 0
delta = Time.new.year - i
if year.length.zero?
  print "Et ton année de naissance ?\n"
else if i == Time.new.year
  print "Tu es trop jeune !\n"
else
  while i <= Time.new.year
    puts("Il y a #{delta} ans, tu avais #{x} ans")
    i += 1
    x += 1
    delta -= 1
  end
end
end
